docs/user-guide/src/introduction.md

```diff
@@ -4,12 +4,50 @@
 
 # Metal³
 
-Metal³ project (pronounced: Metal Kubed) exists to provide components
-that allow you to do bare metal host management for Kubernetes. Metal³
-works as a Kubernetes application, meaning it runs on Kubernetes and is
-managed through Kubernetes interfaces.
+The Metal³ project (pronounced: "Metal Kubed") provides components for bare
+metal host management with Kubernetes. You can enrol your bare metal machines,
+provision operating system images, and deploy Kubernetes clusters to them. From
```

Comment:
Just a  thought - it would be cool if we could mention that the part for provisioning OS images and the part for deploying k8s clusters are separate components that work together, and you can also use the former without the latter. Not a show-stopper but if there's a way to work that in without interrupting the flow.